src/components/layouts/FilterBar.tsx

```diff
@@ -114,15 +114,15 @@ export const FilterBar = ({
         }}
       >
         <AppMargin size={SizeofAppMargin.LARGE} py="14px">
-          <Stack direction="row" alignItems="center" justifyContent={'space-between'}>
-            <Stack direction="row" columnGap={3} alignItems={'center'}>
+          <Stack direction={'row'} justifyContent={'space-between'}>
+            <Stack direction={'row'} columnGap={3}>
```

Comment:
FilterBar should contain border bottom but it seems missing here. While in the production branch, border has been inserted all the sides (which is also not correct according to the design). Here, in this PR border is missing. 

Also, the padding of the Filter bar seems not matching to me. Please kindly correct them.

SS from this PR - 

![Screenshot from 2024-07-08 16-18-08](https://github.com/copilot-platforms/tasks-app/assets/38468429/dce2bb7a-7ac9-4df1-a687-fce75577e0d0)